tools/check_links.py

```diff
@@ -66,12 +79,14 @@ def check_docs(dir_name):
 
 @click.command()
 @click.argument('dirs', type=str, nargs=-1)
-def main(dirs):
+@click.option('-e', '--external', is_flag=True, default=False,
+              help='Whether to test external links.')
+def main(dirs, external):
```

Comment:
nit: can you also provide a default in the function arg?

Reply:
Done